Web.R.tex

```diff
@@ -179,26 +195,27 @@ \section{Goals of the distributed Web}\label{sec:goals}
+To prevent such attacks, it is fundamental to assume a weaker, Byzantine model.
+The previous example is prevented by HTTPS, the more secure Web protocol, which introduces encryption and authentication of transmitted data.
+If the data is altered by a malicious third party during transmission, the client or the Web server can detect it and react accordingly.
 
-HTTPS, the more secure Web protocol, is built around this weaker scenario. It introduces encryption and authentication of transmitted data, which allows it to be transferred over untrusted communication channels. If the data is altered by a malicious third party during transmission, the client or the Web server can detect it and react accordingly.
-
-But this is not the only entity that we need to move to Byzantine behavior. Consider this: if a content creator publishes a website on a web hosting service, that platform is technically able to delete or alter any file of that website, effectively sending to clients different information than the one intended by the author. HTTPS would not be able to protect against this attack, since it only protects the communication channel and not the website files themselves, which are encrypted by the web server only when transmitting them to a client. Also, the Web server is able to send whatever content it desires independently from the files (or refuse to serve content at all), since the client has no way of verifying that the content is as intended by its author.
-
-If we move Web servers from a stopping failure model to a Byzantine one and implement appropriate safeguards such that data can be distributed by an untrusted entity, then we don't need to rely on some specific and trusted system: \emph{anyone} can distribute it.
-
-We can therefore build a distributed storage system where content is shared by a network of servers that anyone can provide. We don't need to trust some hosting service provider with the website data, we can distribute our data to any of these servers and we will be sure that the information is going to be transmitted safely without modifications.
+Simply using HTTPS does not prevent other kinds of attack though. Consider this: if a content creator publishes a website on a web hosting service, that platform is technically able to delete or alter any file of that website, effectively sending to clients different information than the one intended by the author. HTTPS would not be able to protect against this attack, since it only protects the communication channel and not the website files themselves, which are encrypted by the web server only when transmitting them to a client. Also, the Web server is able to send whatever content it desires independently from the files (or refuse to serve content at all), since the client has no way of verifying that the content is as intended by its author.
+As such, web servers must also be considered Byzantine.
+The primary function of a web server is storing and serving content.
```

Comment:
@pacheco This last line seems out of place. What's the purpose of this sentence? I'm not really understanding it.

Reply:
I expanded on this a bit, you can check my version on commit c688c2ee06710cae42fba8bc2c9b8184684860d4, let me know what you think of that

Reply:
The sentence was just a link from webservers to storage systems.
Your version is good, but I suggest changing it slightly, because right now it sounds as if "being byzantine" allows for anyone to share, where it's _handling_ byzantine servers which allow that.
How about the following:

...
As such, web servers must also be considered Byzantine. 
In this chapter, we explore projects which provide distributed storage while tolerating Byzantine servers, allowing for content to be distributed by a network of untrusted entities.
...

Reply:
Actually, I just noticed the sentence that follows already says the same thing.
I suggest just cutting this part. It would become simply:

...
As such, web servers must also be considered Byzantine.
In this chapter, we look into distributed storage systems which allow for data to be safely stored and distributed by untrusted entities, removing the need for trusting a single hosting service provider.

Reply:
Ok, I will keep this last version, I'll just change `distributed by untrusted entities` with `distributed by a network of untrusted entities`.